modules/api/src/main/java/org/apache/fluo/api/observer/MetricsReporter.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.apache.fluo.api.observer;
+
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Reports application metrics using Fluo metrics reporters configured by 'fluo.metrics.reporter.*'
+ * properties. Use unique names when naming metrics.
+ *
+ * @since 1.0.0
+ */
+public interface MetricsReporter {
+
+  /**
+   * Updates counter with change
+   *
+   * @param name Counter name
+   * @param change Positive or negative number to increment/decrement number
+   */
+  void updateCounter(String name, long change);
```

Comment:
Do different metrics need unique names?  Is it ok for a counter and meter to have the same name?